_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+  theme_minimal() +
+  scale_fill_manual(values = poll_colors) +
+  theme(
+    # removes grid lines from plot ----
+    panel.grid = element_blank(),
+    # moves legend to top instead of side ----
+    legend.position = "top",
+    # removes title from legend, often superfluous ----
+    legend.title = element_blank(),
+    # creates the light gray box around the plot, may or may not want this ----
+    #panel.background = element_rect(color = "#F2F2F2"),
+    # creates the gray background boxes for faceting labels ----
+    strip.background = element_rect(
+      color = "#F2F2F2",
+      fill = "#F2F2F2"
+    ),
+    # this produces a fully left justified title ----
+    plot.title.position = "plot",
+    axis.text.x = element_blank(),
+    axis.title.x = element_blank(),
+    #axis.text.y = element_blank(),
+    axis.title.y = element_blank(),
+    )
+```
+
+
+```{r}
+girafe(ggobj = p,
+       #opts_tooltip(use_fill = TRUE)
+       #opts_tooltip(css = tooltip_css, use_fill = TRUE)
+       )
+```
+
+
+
+# Styling content
+
+Additionally, I considered how to visually style content beyond text in a tweet. I asked on [R-Ladies slack](https://rladies-community-slack.herokuapp.com/){target="_blank"} about R-Ladies styles, and I was directed to the xaringan R-Ladies [css](https://github.com/yihui/xaringan/tree/master/inst/rmarkdown/templates/xaringan/resources){target="_blank"} and the R-Ladies [branding](https://guide.rladies.org/organization/tech/brand/){target="_blank"} guides. You are not required to use R-Ladies style and branding, but it was convenient for me. 
+
+I developed two visual layouts using the Google slide [template](https://docs.google.com/presentation/d/1sriC2biLPYza_TtGiZkrNDsv3AMg6dvhqw2yit4wNnA/edit){target="_blank"} from R-Ladies branding (see tweets for [blogdown vs dstill](https://twitter.com/WeAreRLadies/status/1363144545677017089){target="_blank"} and [asking for help online](https://twitter.com/WeAreRLadies/status/1362370580708790274){target="_blank"}).
+
+```{r echo=FALSE, fig.cap="Comparison of blogdown vs distill styled using R-Ladies Google slide template."}
+knitr::include_graphics("img/compare_blogdown_distill.svg")
+```
+
+
+I also created five R-Ladies styled code gifs with xaringan and flipbookr - methods and code are in this [blog post](https://www.pipinghotdata.com/posts/2021-03-08-r-ladies-styled-code-gifs-with-xaringan-and-flipbookr/){target="_blank"}. Here is an example code gif:
+
+```{r echo=FALSE, fig.cap="Example R-Ladies styled code gif."}
+knitr::include_graphics("img/demo-walrus-flipbook-quality.gif")
+```
+
+
+
+
+# First and last tweets
+
+The introduction and farewell tweets as a curator are important as this is when you actually tell people your name or personal twitter handle. I have seen 
+many excellent curators fall short on engagement (likes and re-tweets) with their first and last tweets, resulting in less than merited recognition for the wonderful curator. 
```

Comment:
I think I'd avoid this sentence as engagement could depend on other things (e.g. did rladiesglobal forget to retweet these tweets :see_no_evil:) and it could sound a big judgemental.

Reply:
very good call, thank you! I removed that sentence, hopefully this suffices https://github.com/shannonpileggi/pipinghotdata_distill/pull/2/commits/62bb5aea358a086ea45ee0304338257cbb8d13f5